assets/scss/default.scss

```diff
@@ -0,0 +1,70 @@
+@import "bootstrap";
+
+.row {
+	margin: 20px auto;
+}
+
+figure img, .posters img {
+	max-width: 100%;
+}
+
+hr {
+	border-top: 1px solid #fff;
+}
+
+header {
+	&, a {color: $gray-light;}
+	h1, h2, p {
+		text-transform: uppercase;
```

Comment:
Properties should be ordered letter-spacing, text-transform<br>Each selector in a comma sequence should be on its own single line